src/Data/Sum/Relation/Binary/Pointwise.agda

```diff
@@ -93,6 +94,24 @@ drop-inj₂ (inj₂ x) = x
 ⊎-irreflexive irrefl₁ irrefl₂ (inj₁ x) (inj₁ y) = irrefl₁ x y
 ⊎-irreflexive irrefl₁ irrefl₂ (inj₂ x) (inj₂ y) = irrefl₂ x y
 
+-- need to be more explicit here otherwise Agda does something odd with the variables
+⊎-wellFounded : {A : Set a} {≈₁ : Rel A ℓ₁} {≈₂ : Rel A ℓ₂} → WellFounded ≈₁ → WellFounded ≈₂ → WellFounded (Pointwise ≈₁ ≈₂)
+⊎-wellFounded {≈₁ = ≈₁} {≈₂} wf₁ wf₂ x = acc (⊎-acc x)
```

Comment:
Can fix both the comment, and the `variable`s:
```suggestion
⊎-wellFounded : WellFounded ≈₁ → WellFounded ≈₂ → WellFounded (Pointwise ≈₁ ≈₂)
⊎-wellFounded {≈₁ = ≈₁} {≈₂ = ≈₂} wf₁ wf₂ x = acc (⊎-acc x)
```

Reply:
Fixed.